en/services/illuminator.md

```diff
@@ -0,0 +1,154 @@
+# Illuminator Protocol
+
+## Introduction
+
+The illuminator protocol allows MAVLink control over the behavior of lights, LEDs, and/or emitters mounted or integrated on the drone.
+The protocol currently allows for the following control: brightness, on/off, and a strobe feature.
+
+Along with this, the illuminator protocol also publishes status information for developers or users. The status messaging encompasses the current configuration of the illuminator and the health of the illuminator device.
+
+## Concepts

```

Comment:
I would remove this heading, and move the next two headings up a level. 

Then put a new heading above "### Illuminator Connection" something like `## Usage` or `## Sequences`